### ISSUE TITLE
Where is v0.6.0 in rubygems?

### ISSUE THREAD

**Issue by drnic** [marcoroth/turbo_power#128]
There is latest v0.6.0 release and I try to request it in Gemfile:

```ruby
gem "turbo_power", "~> 0.6.0"
```

But its not on rubygems https://rubygems.org/gems/turbo_power ? Did it get yanked or not uploaded?

```
$ bundle update
Could not find gem 'turbo_power (~> 0.6.0)' in rubygems repository https://rubygems.org/ or installed locally.

The source contains the following gems matching 'turbo_power':
  * turbo_power-0.1.0
  * turbo_power-0.1.1
  * turbo_power-0.1.2
  * turbo_power-0.1.3
  * turbo_power-0.1.5
  * turbo_power-0.1.6
  * turbo_power-0.2.0
  * turbo_power-0.3.0
  * turbo_power-0.3.1
  * turbo_power-0.4.0
  * turbo_power-0.5.0
```

**Comment by drnic** [comment]
Ugh. I've gotten confused. I assumed there'd be matching versions for npm + rails.

**Comment by marcoroth** [comment]
Thanks for reporting! There should be matching versions, let me check what's up here!

**Comment by marcoroth** [comment]
I just published the missing v0.6.0 to RubyGems, sorry for the confusion!
